bin/llvm-krun

```diff
@@ -347,7 +347,21 @@ else
 fi
 
 if $dryRun; then
-  if [ -n "$real_output_file" ]; then
+  if [ $binary_input != $binary_output ] && [ -n "$real_output_file" ]; then
+    (
+    if [ -n "$verbose" ]; then
+      set -x
+    fi
+    "$(dirname "$0")/kore-convert" "$expanded_input_file" > "$real_output_file"
```

Comment:
```suggestion
    "$(dirname "$0")/kore-convert" "$expanded_input_file" -o "$real_output_file"
```

This should use `-o` rather than a redirect; `kore-convert` doesn't have a check for whether `stdout` is a TTY yet so binary output will break this. This is a limitation of the tool API, really